### DISCUSSION TITLE
Products Added To Child Collection Should Also Be Placed In Parent(s)

### DISCUSSION THREAD

**Discussion by poppabear8883** [lunarphp/lunar#537] [Feature Requests]
I think that when adding a `Product` to a *Child* collection it should automatically be placed in the parent(s) ...

For example:

Say I have a *Parent Collection* `Signs` and that collection has a *Child Collection* `Gaming & Streaming`.

And I add a *Product* to the *Child Collection* `Gaming & Streaming`, but maybe I want to view all products from the parent and all of its children.

We can do this by adding the products to both collections, but I feel like this is such a common structure that maybe its a default behavior ?

What are your guys thoughts ?

**Comment by glennjacobs** [comment]
I think that behaviour wouldn't be intuitive. 

By adding a custom observer you should be able to achieve this. 

But I would suggest this is more of a display issue. We have the data in the database, we just need a nice way to get all products from a category and its descendants.
